docs/2.8.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -378,8 +383,9 @@ must be unique.
 Therefore, the protocol upgrade process boils down to:
 
 - Deploy a new domain next to the old domain. Ensure that the new domain is using the desired protocol version.
-  Ensure that you are using different databases (or at least different schemas in the same database), channel names, smart contract addresses etc.
-  It must be a completely separate domain (albeit you can reuse your DLT backend as long
+  Ensure that you are using different databases (or at least different schemas in the same database)
+  for all the domain nodes (mediator, sequencer node and topology manager), channel names, smart contract addresses etc.
+  It must be a completely separate, new domain (albeit you can reuse your DLT backend as long
   as you use different sequencer contract addresses or Fabric channels).
```

Comment:
```suggestion
  as you use different sequencer contract addresses or Fabric channels.
```

(apply this change only if you apply the preceding change that removes the open paren)